feat(gnokey): --gas-wanted=auto --gas-fee=auto (#4451)
gnolang/gno: docs/users/interact-with-gnokey.md

- `-pkgdir` - local path where your is located
- `-broadcast` - enables broadcasting the transaction to the chain
- `-deposit` - a deposit amount of GNOT to send along with the transaction
- `-gas-wanted` - the upper limit for units of gas (optional, defaults to auto)

> - `-gas-fee` - amount of GNOTs to pay per gas unit (optional, defaults to auto)

Reviewer: ```suggestion
- `-gas-fee` - amount of GNOTs to pay (optional, defaults to auto)
```
It is not "per gas unit". The `-gas-fee` is the actual number of coins to be spent.

### Gas parameters

Gnokey supports **automatic gas estimation** by default, making transactions 
easier to send without manually specifying gas parameters:

```bash
# Automatic gas estimation (default)
gnokey maketx call \
  --pkgpath "gno.land/r/demo/boards" \
  --func "CreateBoard" \
  --args "MyBoard" "Board description" \
  --remote https://rpc.gno.land:443 \
  --chainid staging \
  YOUR_KEY_NAME
```

#### Manual Gas Control (Optional)

For advanced users who want to control gas parameters manually, you can 
specify:

- `--gas-wanted` - Maximum gas units to consume (or "auto" for estimation)

> - `--gas-fee` - Fee per gas unit (or "auto" for automatic pricing)

Reviewer: ```suggestion
- `--gas-fee` - Amount to pay (or "auto" for automatic pricing)
```
Same comment as above. It is not "per gas unit". The `-gas-fee` is the actual number of coins to be paid.
